scripts/exglobal_prep_land_obs.py

```diff
@@ -17,7 +17,10 @@
 
     # Take configuration from environment and cast it as python dictionary
     config = cast_strdict_as_dtypedict(os.environ)
+    HH = os.environ.get("CDATE")[8:]
```

Comment:
isn't there `cyc`?

Reply:
@aerorahul is going to have a preferred solution for this.